src/sgx/faasm_sgx_attestation.cpp

```diff
@@ -99,11 +112,15 @@ extern "C"
         faasm_sgx_status_t ret_val;
         uint32_t cb_store_id;
 
+        auto logger = faabric::util::getLogger();
+
         if ((ret_val = _find_callback_store_slot(
                &cb_store_id, faasletSgxMsgBufferPtr)) != FAASM_SGX_SUCCESS) {
             return ret_val;
         }
 
+        logger->debug("sending message to key manager ({})", msg->msg_id);
```

Comment:
The use of capitals in this PR and even this file is inconsistent. Log messages should start with a capital letter